index.html

```diff
@@ -69,7 +69,384 @@ <h4 class="sub-title">I am a Software Developer</p>
 
       </section>
 
+  
+      <section class="wrap">
+        
+        <header class="main-background">
+          <h1 class="main">My Recent Works</h1>
+       </header>  
+  
+       <hr class="line" style="width:327px",  color: #212121; >  
+       
+       <main>
+
+        <div class="img"><img src="./images/placeholder.png" alt=""></div>
+  
+        <h2 class="heading2">Multi-Purpose Stories</h2>
+  
+          <div>
+      <p class="text">
+        A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.
+      </p>
+      </div>
+  
+      <div class="image"></div>
+
+      <div class="prog-button">
+  
+      <ul class="button2">
+          <li class="css">css</li>
+          <li class=" html">html</li>
+          <li class="bootstrap">bootstrap</li>
+          <li class="ruby">ruby</li>
+          <li class="see">See Project</li>
+    </ul>
+      </div>
+        
+    </main>
+      
+    </section>
+
+    <section class="wrap27">
+
+    
+    <header class="prof-art1">
+      <h1 class="art99">Professional Art</h1>
+      <h2 class="data99">Printing Data</h2>
+    </header>
+
+    <Main>
+      <div class="text20">
+      <p>
+        A daily selection of privately
+         personalized reads; no accounts
+          or sign-ups required. has been the
+           industry's standard
+      </p>
+    </div>
+    
+      <div class="program">
+
+        <ul class="prog-item22">
+          <li class="html111">html</li>
+          <li class="boot110">bootstrap</li>
+          <li class="ruby12">ruby</li>
+        </ul>
+      
+        <div class="see122">See Project</div>
```

Comment:
- [ ] Kindly position this div and give it appropriate color like what is on the guiding template. It is not properly positioned at this moment.

![button-reposition](https://user-images.githubusercontent.com/26597432/199763870-a08fc880-410b-4bf6-9807-a7b3e804c467.png)


- [ ] Kindly make the div and link or better still change it to `button` for semantic reason.